x/stakeibc/keeper/unbonding_records_get_host_zone_unbondings_msgs_test.go

```diff
@@ -888,3 +912,140 @@ func (s *KeeperTestSuite) TestGetUnbondingICAMessages() {
 		})
 	}
 }
+
+func (s *KeeperTestSuite) TestConsolidateUnbondingMessages_Success() {
+	batchSize := 4
```

Comment:
Do we need tests for edge cases with o-weight vals?

Reply:
Good question! It's actually covered by [this](https://github.com/Stride-Labs/stride/pull/1018/files#diff-42324da485a3df105dabdb479a72a8b7e82535250b0742a415e1c590081913ffR1023)

Zero weight validators will have their capacity equal to their current delegation ([example](https://github.com/Stride-Labs/stride/pull/1018/files#diff-42324da485a3df105dabdb479a72a8b7e82535250b0742a415e1c590081913ffR1031))